examples/using-nested-routes/elements/selection.js

```diff
@@ -0,0 +1,8 @@
+const Tram = require('../../../tram-one')
+const html = Tram.html()
+
+module.exports = (attrs, children) => html`
+  <div style="display: flex; justify-content: space-around; padding: 0.5em;">
```

Comment:
I personally would prefer this to be pulled out into its own little styling area but just my opinion

Reply:
eh, this element isn't really anything except that styling. At that point, I would be tempted to just have the CSS string as a variable in the parent page...

Reply:
Then do that?

Reply:
eh, I like this as another example of using registered elements.